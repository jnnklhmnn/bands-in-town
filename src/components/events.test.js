import React from 'react';
import Events from './Events.js';

import renderer from 'react-test-renderer';

test('Events should match Snapshot', () => {

    const mockinfo = {
        name: 'foo',
        thumb_url: 'http://foo.bar.de'
    }

    const mockevents = [
        {
            "id": "foo",
            "datetime": "2018-08-04T20:00:00",
            "venue": {
                "name": "The Theater at MGM National Harbor",
                "latitude": "38.791707",
                "longitude": "-77.003002",
                "city": "National Harbor",
                "region": "MD",
                "country": "United States"
            }
        },
        {
            "id": "bar",
            "datetime": "2018-08-05T20:00:00",
            "venue": {
                "name": "The Theater at MGM National Harbor",
                "latitude": "38.791707",
                "longitude": "-77.003002",
                "city": "National Harbor",
                "region": "MD",
                "country": "United States"
            }
        }
    ];

    const component = renderer.create(
        <Events info={mockinfo} events={mockevents} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});