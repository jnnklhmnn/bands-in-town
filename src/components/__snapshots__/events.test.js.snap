// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Events should match Snapshot 1`] = `
<div
  className="contentbox"
>
  <div
    className="headerwrapper"
  >
    <h3
      className="name"
    >
      Go See 
      foo
       live!
    </h3>
    <img
      alt="foo"
      className="image"
      src="http://foo.bar.de"
    />
  </div>
  <ul
    className="list"
  >
    <li
      className="listitem"
    >
      <span
        className="left"
      >
        <p
          className="listitemcopy"
        >
          <span
            className="month"
          >
            aug
          </span>
           
          <span
            className="day"
          >
             
            04
          </span>
        </p>
      </span>
      <span
        className="right"
      >
        <p
          className="listitemcopy"
        >
          The Theater at MGM National Harbor
        </p>
        <p
          className="listitemcopy"
        >
          National Harbor
        </p>
        <p
          className="listitemcopy"
        >
          United States
        </p>
      </span>
    </li>
    <li
      className="listitem"
    >
      <span
        className="left"
      >
        <p
          className="listitemcopy"
        >
          <span
            className="month"
          >
            aug
          </span>
           
          <span
            className="day"
          >
             
            05
          </span>
        </p>
      </span>
      <span
        className="right"
      >
        <p
          className="listitemcopy"
        >
          The Theater at MGM National Harbor
        </p>
        <p
          className="listitemcopy"
        >
          National Harbor
        </p>
        <p
          className="listitemcopy"
        >
          United States
        </p>
      </span>
    </li>
  </ul>
</div>
`;
